README.md

```diff
@@ -1,3 +1,6 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+## Local setup
+
+- To run locally you will need to add two .env file which will connect to your local DBs, one for testing and one for dev-data. Add these two files in /db, naming convention is .env.[TESTDBNAME] and .env.[DEVDBNAME].
+- You can then use the scripts to setup your local PSQL DB by running npm setup-dbs.
```

Comment:
Hey Henry, the .env files should not contain the database name in their name. That would be inside them. Please correct your instructions and also include what those files should contain.